tools/assetpipelineeditor/main.qml

```diff
@@ -81,9 +81,15 @@ Kuesa.SceneEntity {
             }
         },
         ObjectPicker {
+            property point __downPos
+            onPressed: __downPos = pick.position
             onClicked: {
+                if (pick.position !== __downPos)
```

Comment:
In C++, __variable are reserved for implementation. Im not used to QML and if is typical to name properties this way. If not, maybe is better to change the name?

Reply:
it's the same idea, it's a convention to indicate the property is reserved for the item's internal implementation and should not be be used from outside